src/components/Pager/Pager.tsx

```diff
@@ -59,31 +62,43 @@ const Pager = ({ pageSize, totalRecords, onPageChange }: PagerProps) => {
   };
 
   return (
-    <div className="flex space-x-1 text-[#73808C] text-sm">
+    <div className="flex space-x-1 text-[#73808C] text-sm justify-end">
       <div>
         {fromRecord}-{toRecord}
       </div>
       <div>of</div>
       <div>{totalRecords}</div>
-      <div className="ml-2">
-        <img
-          className={classNames('h-3 w-3 mt-1', {
+      <div>
+        <svg
+          className={classNames('h-3 w-3 mt-1 ml-1 stroke-[#8F99A3]', {
             'cursor-pointer': currentPage > 1,
+            'hover:stroke-[#454D54]': currentPage > 1,
           })}
-          src={pagePreviousIcon}
-          alt={'Previous Page'}
+          width="7"
+          height="13"
+          viewBox="0 0 7 13"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
           onClick={() => decrementPageNumber()}
-        />
+        >
+          <path d="M6.5 0.5L0.5 6.5L6.5 12.5" stroke-linecap="round" stroke-linejoin="round" />
+        </svg>
```

Comment:
While this approach of directly embedding the SVG content works, it's not the most ideal practice.

In general, it's preferable to keep assets as separate files, and only use the svg code directly for uncommon modifications (e.g., altering path values for animation purposes).

For this case, I recommend continuing to import the SVG from the files and applying the fill class after the import. You can find more information on using the fill class in the [Tailwind CSS documentation](https://tailwindcss.com/docs/fill).

By following this approach, we can maintain a cleaner and more organised codebase.

---

Please feel free to ping me in Slack if you've any question!

Reply:
@Publicker I can't seem to get the 'hover:stroke' working when using an 'img' as opposed to an 'svg'. When you're free can you give me a shout to discuss?